### DISCUSSION TITLE
Esperanto Translation

### DISCUSSION THREAD

**Discussion by watanuki92** [ChristopherKlay/StadiaEnhanced#165] [Translations]
LANG: eo

var translation = {
    default: 'Defaŭta',
    native: 'Origina',
    hide: 'Kaŝi',
    show: 'Montri',
    total: 'Totalo',
    visible: 'Videbla',
    hidden: 'Kaŝita',
    enabled: 'Ŝaltita',
    disabled: 'Malŝaltita',
    auto: 'Aŭtomata',
    manual: 'Permana',
    all: 'Ĉio',
    locked: 'Barita',
    complete: 'Kompleta',
    incomplete: 'Malkompleta',
    games: 'Ludoj',
    allgames: 'Ĉiuj ludoj',
    leavepro: 'Forlasas Pro',
    bundles: 'Faskoj',
    addons: 'Aldonaĵoj',
    wishlist: 'Dezirlisto',
    responsive: 'Sinadapta',
    windowed: 'Fenestra',
    fullscreen: 'Plenekrana',
    onsale: 'Rabatoj',
    prodeals: 'Rabatoj de Stadia Pro',
    userprofile: 'Mia profilo',
    usermedia: 'Captures & game states',
    searchbtnbase: 'Serĉi en',
    avatarpopup: 'Nova profilbilda URL (Lasu ĝin malplenan por defaŭta ):',
    sessiontime: 'Tempo de seanco',
    codec: 'Kodeko',
    resolution: 'Distingivo',
    hardware: 'Hardvaro',
    software: 'Softvaro',
    trafficsession: 'Session traffic',
    trafficcurrent: 'Current traffic',
    trafficaverage: 'Average traffic',
    packetloss: 'Perditaj paketoj',
    framedrop: 'Frames dropped',
    latency: 'Respondotempo',
    jitter: 'Jitter Buffer',
    decodetime: 'Decoding Time',
    compression: 'Densigo',
    bitrate: 'Bitrapido ',
    streammon: 'Stream Monitor',
    stream: 'Elsendo',
    network: 'Reto',
    session: 'Seanco',
    extdetail: 'Pliaj detaloj',
    maxresolution: 'Maximuma distingivo',
    fps4K: 'Framerate @ 4K',
    testdiscl: '<b>Disclaimer:</b> This game has yet to be tested.',
    datadiscl: 'This is the maximum framerate achieved when playing a game in 4K mode (requires Stadia Pro).\
    On games with a resolution/framerate toggle, resolution was picked. \
    This data is provided by <a href="https://twitter.com/OriginaIPenguin" target="_blank">@OriginaIPenguin</a> \
    and the full database can be found <a href="https://linktr.ee/StadiaDatabase" target="_blank">here</a>.',
    noteOne: '4K Mode',
    noteTwo: '30/60 FPS Toggle',
    noteThree: '60 FPS in 1080p mode',
    noteFour: '30 FPS in 1080p Mode',
    noteFive: 'Not compatible with 4K mode',
    unsupported: 'Unsupported',
    crossfriends: 'No Cross-platform Buddy System',
    community: 'Komunumo',
    speedtest: 'Speedtest',
    quickaccess: 'Rapida aliro',
    messages: 'Mesaĝoj',
    comfeature: "Komunumaj funkcioj",
    avatar: 'Profilbildo',
    interface: 'Interfaco',
    shortcut: 'StadiaIcons',
    shortcuttitle: 'Krei ŝparvojon por',
    shortcutdesc: 'Ebligas krei ŝparvojon por ludo en via aparato.',
    stadiastats: 'StadiaStats',
    stadiastatsopen: 'Vidi je StadiaStats.GG',
    stadiastatsdesc: 'Enables direct shortcuts to game statistics, link to your profile and the find-a-buddy system on stadiastats.gg.',
    gridsize: 'Grid Size',
    griddesc: 'Changes the amount of games per row in the library.',
    clock: 'Horloĝo',
    clockdesc: 'Displays the current time on the friends list, as a in-game overlay, or both.',
    friendslist: 'Listo de amikoj',
    igoverlay: 'Enluda plustavolo',
    listoverlay: 'List & Overlay',
    filter: 'Game Filter',
    filterdesc: 'Allows you to sort your library by hiding games. The filter can be toggled via the symbol, top-right above your games in the library.',
    invitebase: 'Copy invite link',
    inviteactive: 'Kopiita!',
    gamelabel: 'Game Labels',
    gamelabeldesc: 'Removes labels like "Pro" from games on the homescreen.',
    homegallery: 'Uzanta galerio',
    homegallerydesc: 'Hides the "Captures" area at the bottom of the homescreen.',
    quickprev: 'Antaŭrigardo de mesaĝo',
    quickprevdesc: 'Hides the message preview in the friends list.',
    quickrep: 'Tujrespondo',
    quickrepdesc: 'Hides the quick reply option in chats.',
    offlinefriend: 'Elretaj amikoj',
    offlinefrienddesc: 'Hides offline friends in the friends list.',
    invisiblefriend: 'Invisible Friends',
    invisiblefrienddesc: 'Hides friends with unknown online status in the friends list.',
    notification: 'Sciigoj',
    notificationdesc: 'Montras sciigon kiam Stadia Enhanced ĝisdatiĝas al nova versio("Aŭtomata" kaŝas post 5 sekundoj, "Permana" atendas agon de la uzanto).',
    streammodedesc: 'Ŝaltu por igi kelkajn elementojn (ekzemple: la liston de amikojn) nelegeblajn dum vi elsendas (pere de iloj kiel OBS / Discord).',
    streammode: 'Elsenda reĝimo',
    catprev: 'Category Preview',
    catprevdesc: 'Hides the category tags when hovering over a game.',
    streammondesc: 'Activate to start the monitor whenever a game starts.',
    resolutiondesc: 'The targeted resolution for game streams. 1440p and 2160p require VP9 and Stadia Pro.',
    codecdesc: 'The codec used for game streams.',
    confirmreset: 'Are you sure you want to reset the settings?',
    statistics: 'Statistikoj',
    gamesfinished: 'Finitaj ludoj',
    achievementsunlocked: 'Achievements Unlocked',
    totalPlayTime: 'Totala ludtempo',
    splitstore: 'Split Store Lists',
    splitstoredesc: 'Splits store lists into two columns for a better overview.',
    inlineimage: 'Antaŭmontri bildon',
    inlinedesc: 'Replaces image links for common file formats (jpg/gif/png) with a clickable preview.',
    familyelements: 'Family-sharing options',
    familyelementsdesc: 'Hides the "Share this game with family" options.',
    donations: 'Donacoj',
    reportbug: 'Cimraporti',
    exportset: 'Export settings',
    importset: 'Import settings',
    importerror: 'The file you are trying to open does not contain a valid Stadia Enhanced profile.',
    resetsettings: 'Reset settings'
}

**Comment by ChristopherKlay** [comment]
Thanks for the work on this!

Google (so far) sadly doesn't support the language for Stadia, but the translation for the extension will be available with the next update (:

**Reply by watanuki92** [reply]
You are welcome :) the translation is not completely finished but the main UI items are translated already. Even if Google doesn't support the language, the text should be displayed in Esperanto as long as you select it from the dropdown menu, right? :). 

Also: How can I submit changes to my translation? Just so I can finish translating it.

**Reply by ChristopherKlay** [reply]
> Even if Google doesn't support the language, the text should be displayed in Esperanto as long as you select it from the dropdown menu, right? :)

Correct! If you set Esperanto as your main language in Googles settings, it *might* actually default to english, while still loading the translation by default as well. Never tested that however.

> Also: How can I submit changes to my translation? Just so I can finish translating it.

Absolutely, at any time just edit the original post and comment with a small note, or leave the changes in a new comment on their own. They will get implemented with the next patch (:
